### PR TITLE
add sol

### PR DESCRIPTION
**Pull request by bohdanolya**
add field and get set

### CODE REVIEW

File: src/main/java/ua/com/ushop/entity/Product.java [hunk 1]
```diff
@@ -15,10 +15,19 @@
     private int id;
     private String title;
     private String file;
+    private String Sol;
 
     public Product() {
     }
 
+    public String getSol() {
```
Comment (bohdanolya) [inline src/main/java/ua/com/ushop/entity/Product.java]: why?

File: src/main/java/ua/com/ushop/entity/Product.java [hunk 2]
```diff
@@ -15,10 +15,19 @@
...
+        return Sol;
+    }
+
+    public void setSol(String sol) {
```
Comment (bohdanolya) [inline src/main/java/ua/com/ushop/entity/Product.java]: ??????

Comment (solyamia) [reply src/main/java/ua/com/ushop/entity/Product.java]: fffffff

File: src/main/java/ua/com/ushop/entity/Product.java [hunk 3]
```diff
@@ -15,10 +15,19 @@
...
     private String title;
     private String file;
+    private String Sol;
 
     public Product() {
     }
 
+    public String getSol() {
+        return Sol;
+    }
```
Comment (solyamia) [inline src/main/java/ua/com/ushop/entity/Product.java]: ll

File: src/main/java/ua/com/ushop/entity/Product.java [hunk 4]
```diff
@@ -16,9 +16,19 @@
     private String title;
     private String file;
 
+    private String Solomiya;
```
Comment (bohdanolya) [inline src/main/java/ua/com/ushop/entity/Product.java]: ok

**Pull request merged**